model/user/user.go

```diff
@@ -357,11 +357,24 @@ func (u *DBUser) GetViewableProjectSettings(ctx context.Context) ([]string, erro
 	}
 	roleManager := evergreen.GetEnvironment().RoleManager()
 
-	viewProjects, err := rolemanager.FindAllowedResources(ctx, roleManager, u.Roles(), evergreen.ProjectResourceType, evergreen.PermissionProjectSettings, evergreen.ProjectSettingsView.Value)
+	viewableProjects, err := rolemanager.FindAllowedResources(ctx, roleManager, u.Roles(), evergreen.ProjectResourceType, evergreen.PermissionProjectSettings, evergreen.ProjectSettingsView.Value)
 	if err != nil {
 		return nil, err
 	}
-	return viewProjects, nil
+	return viewableProjects, nil
+}
+
+func (u *DBUser) GetViewableProjects(ctx context.Context) ([]string, error) {
```

Comment:
This looks great! should just add a comment, and a unit test (similar to https://github.com/evergreen-ci/evergreen/blob/bddd8d0eebd14ba2db628d9cf3ef86d699f237ea/model/user/user_test.go#L757-L756)